src/fundus/parser/data.py

```diff
@@ -139,6 +138,8 @@ def to_unicode_characters(text: str) -> str:
             self.__xml = lxml.etree.fromstring(xml)
         return self.__xml
 
+    __value_regex__ = re.compile("^<[^<]*>(?P<value>.*)<[^<]*>$", flags=re.DOTALL)
```

Comment:
```suggestion
    __value_regex__ = re.compile("^<[^<]*>(?P<value>.*)</[^<]*>$", flags=re.DOTALL)
```
Nitpick, but perhaps for consistency, we would expect the second tag to be a closing tag.